eras/conway/impl/CHANGELOG.md

```diff
@@ -1,7 +1,11 @@
 # Version history for `cardano-ledger-conway`
 
-## 1.13.0.0
-
+	## 1.13.0.0
+	
+* Changed GovActionState to have 1 field (gasProposalProcedure) rather than 3 (gasDeposit,gasAction,gasReturnAddr)
+  * the old field names (gasDeposit,gasAction,gasReturnAddr) become functions, and the lenses
+  * (gasDepositL,gasActionL,gasReturnAddrL) have the same type, but behave differently.
+  *  Added the lenses: pProcDepositL,  pProcGovActionL,  pProcReturnAddrL,  pProcAnchorL, gasProposalProcedureL.
```

Comment:
```suggestion
* Changed GovActionState to have 1 field (gasProposalProcedure) rather than 3 (gasDeposit,gasAction,gasReturnAddr)
  * the old field names (gasDeposit,gasAction,gasReturnAddr) become functions, and the lenses
  * (gasDepositL,gasActionL,gasReturnAddrL) have the same type, but behave differently.
  *  Added the lenses: `pProcDepositL`, `pProcGovActionL`, `pProcReturnAddrL`,  `pProcAnchorL`, `gasProposalProcedureL`.
```